src/lexer/language.lua

```diff
@@ -441,40 +450,51 @@ return {
 		},
 
 		UDim2 = {
-			new = true,
-			fromScale = true,
 			fromOffset = true,
+			fromScale = true,
+			new = true,
 		},
 
 		Vector2 = {
 			new = true,
-			
+
 			one = true,
-			zero = true,
 			xAxis = true,
 			yAxis = true,
+			zero = true,
 		},
 
 		Vector2int16 = {
 			new = true,
 		},
 
 		Vector3 = {
-			new = true,
 			fromAxis = true,
-			fromNormalId = true,
 			FromAxis = true,
+			fromNormalId = true,
 			FromNormalId = true,
-			
+			new = true,
+
 			one = true,
-			zero = true,
 			xAxis = true,
 			yAxis = true,
 			zAxis = true,
+			zero = true,
 		},
 
 		Vector3int16 = {
 			new = true,
 		},
 	},
 }
+
+-- Filling up language.libraries.Enum table 
+local enumLibraryTable = language.libraries.Enum
+
+for _, enum in ipairs(Enum:GetEnums()) do
+	if (typeof(enum)) == "Enum" then --TODO: Remove brackes around typeof(enum) once typechecker engine recognizes "Enum" as a valid type
```

Comment:
Do we need this check? `:GetEnums()` is guaranteed to return an array of Enums so we can assume the type is correct.

Reply:
Also, if we do need the check I prefer it to be a guard clause instead, keeping the primary code as non-indented as possible.

```Lua
if (typeof(enum)) == "Enum" then continue end
...
```

Reply:
> Do we need this check? `:GetEnums()` is guaranteed to return an array of Enums so we can assume the type is correct.

For some reason I thought it would return the `GetEnums` function itself within that table. That's an oversight on my end. Will fix it when I come back home.

Reply:
Alright, fixed it now.